doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -426,11 +600,22 @@ To launch C application next steps should be performed.
 2. Launch the executable file.
 
         ./kaa-app
-
+        
 </div>
 
-<div id="run-c++" class="tab-pane fade" markdown="1" >
+<div id="run-cpp" class="tab-pane fade" markdown="1" >
+To launch Cpp application next steps should be performed.
```

Comment:
I agree withyou, it must be with correct grammar and must have more sens.
It needs to change in all doc, because I do the same how in the examples. Each part of the doc should be the same.
I think need to fix all doc, because it has some problems with it. 

For example c code was:
To launch C application next steps should be performed.
Note! I don't see coma. But better to write:
To launch the C application, next steps should be performed: